clang/test/ClangScanDeps/response-file-clang-cl.c

```diff
@@ -0,0 +1,36 @@
+// Check that the scanner can adjust arguments by reading .rsp files in advance.
+
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: sed -e "s|DIR|%/t|g" %t/cdb.json.template > %t/cdb.json
+// RUN: echo /Fo%t/tu.obj >> %t/args_nested.rsp
```

Comment:
I tried playing around with response files and they don't play well with newlines. I think that instead of appending to the response files with `echo`, we should use `split-file` and create two separate files. That will also be more declarative and will make it obvious that one contains `/c` and the other `/E`.

Reply:
Can you elaborate please on "I tried playing around with response files and they don't play well with newlines"?
Things we working on my end (empty lines in .rsp files). Also, Unreal Engine generates .rsp files with each argument on a new line, so if you're seeing any issues there we shall fix them.

Reply:
I recall running `%clang_cl` with whatever arguments and response files this test generated and seeing Clang's argument parser hit an assertion while working with the expanded arguments.